SWLOR.Game.Server/Feature/SpawnDefinition/CZ220SpawnDefinition.cs

```diff
@@ -18,14 +20,22 @@ public class CZ220SpawnDefinition: ISpawnListDefinition
 
         private void DroidSpawns(SpawnTableBuilder builder)
         {
+            var deathAnimation = new Animator()
```

Comment:
This all looks great. The only thing I'm on the fence about is the construction of the animator. It kind of goes against the builder pattern I've set up at this point. I was thinking maybe adding another dot method would be more consistent? My goal with the builders was to make it relatively easy to understand what options are available. 

How do you feel about having .PlayAnimation take in the duration, event, and vfx instead of another object?

I.E:

.PlayAnimation(DurationType.Instant, AnimationEvent.CreateOnDeath, VisualEffect.Fnf_Fireball)

Reply:
I leaned away from that approach for two reasons:

1) in case we need to extend with new params (Scale already an example), we wouldn't have to modify contracts or write a new overload for the PlayAnimation method

2) If another animation service besides IAnimator was ever used.  Unlikely, but we get the benefit of contract flexibility with the `IAnimator` as the only parameter.

Ultimately these aren't highly persuasive arguments, so I defer to you - let me know if you want me to change to discrete args and I'll code it up.

Reply:
Those are fair points, tbh. But if it's all the same to you, let's go ahead and make it a discrete method for now for consistency's sake. Should it expand in scope outside of scale (i.e 5 or more args) we can refactor it to take in the object instead. Basically all you need to do is move the object construction inside the builder. All the stuff you've got now is perfect.

Reply:
Sounds good to me. Made that change.